voice/bxml/verbs/startRecording.md

```diff
@@ -9,15 +9,20 @@ All audio on both sides of the call will be recorded until the call ends or the
-| password                     | (optional) The password to send in the HTTP request to `recordingAvailableUrl`. If specified, the URL must be TLS-encrypted (i.e., `https`).                                                                           |
+| transcribe                   | (optional) A boolean value to indicate that recording should be transcribed. Default is `false`.                                                                                                                       |
+| transcriptionAvailableUrl    | (optional) URL to send the [Transcription Available](../callbacks/transcriptionAvailable.md) event to once it has been processed. Does not accept BXML.                                                                |
+| transcriptionAvailableMethod | (optional) The HTTP method to use for the request to `transcriptionAvailableUrl`. GET or POST. Default value is POST.                                                                                                  |
+| username                     | (optional) The username to send in the HTTP request to `recordCompleteUrl`, `recordingAvailableUrl` or `transcriptionAvailableUrl`. If specified, the URLs must be TLS-encrypted (i.e., `https`).                      |
+| password                     | (optional) The password to send in the HTTP request to `recordCompleteUrl`, `recordingAvailableUrl` or `transcriptionAvailableUrl`. If specified, the URLs must be TLS-encrypted (i.e., `https`).                      |
 | tag                          | (optional) A custom string that will be sent with this and all future callbacks unless overwritten by a future `tag` attribute or cleared.<br><br>May be cleared by setting `tag=""`<br><br>Max length 256 characters. |
 | fileFormat                   | (optional) The audio format that the recording will be saved as: `mp3` or `wav`.  Default value is `wav`.                                                                                                              |
 | multiChannel                 | (optional) A boolean value indicating whether or not the recording file should separate each side of the call into its own audio channel. Default value is `false`. `true` results in two channels.                    |
 
 If the `recordingAvailableUrl` attribute is specified, then the [Recording Available](../callbacks/recordingAvailable.md)
 event is sent to the URL once the recording is available for download, indicating the `mediaUrl` and if there was any issue processing the recording.
 
+If the `transcriptionAvailableUrl` attribute is specified, then the [Transcription Available](../callbacks/transcriptionAvailable.md) event is sent to the URL once the transcription is available for download. BXML returned in response to this callback will be ignored.
+
 BXML returned in response to this callback will be ignored.
```

Comment:
This line should be with the above paragraph instead of this one

Reply:
nice catch!